components/advancedSettings/JetpackBoost/InstallActivatePluginButton.js

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from 'react';
+import apiFetch from '@wordpress/api-fetch';
+import { Spinner } from "@newfold/ui-component-library";
+
+const InstallActivatePluginButton = ( { methods, constants, setModuleStatus } ) => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [isActive, setIsActive] = useState(false);
+  const [message, setMessage] = useState(null);
+
+  const handleInstallActivate = async () => {
+    setIsLoading(true);
+    setMessage( constants.text.jetpackBoostInstalling );
+
+    const apiUrl = methods.NewfoldRuntime.createApiUrl("/newfold-installer/v1/plugins/install");
+    const INSTALL_TOKEN = methods.NewfoldRuntime.ecommerce.install_token;
+    const plugin = 'jetpack-boost';
+
+    try {
+      await methods.apiFetch({
+        url: apiUrl,
+        method: "POST",
+        headers: { "X-NFD-INSTALLER": INSTALL_TOKEN },
+        data: { plugin, activate: true, queue: false },
+      });
+      setIsActive(true);
+      setModuleStatus(true);
+      setMessage( constants.text.jetpackBoostInstalling );
+    } catch (error) {
+      console.log(error.message);
```

Comment:
Should this be a `console.error()` instead?

Reply:
That's right, I replaced the function and committed the change